pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "PyYAML",
     "requests",
     "toml",
+    "tidecv",
```

Comment:
This should be in the `engine` group below. (But there is no way for you to have known that).

Reply:
updated